CONTRIBUTING-CSS.md

```diff
@@ -189,18 +188,17 @@ The `title` property is displayed above the editor, and should be of the form: "
 
 The guidance above assumes you're documenting a CSS property. But you can also write examples for CSS functions, like [`linear-gradient()`](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient), or types, like [`angle`](https://developer.mozilla.org/en-US/docs/Web/CSS/angle). If you do this, there are a couple of special considerations.
 
-*   the name of the HTML file you write must be prefixed with `function-` for functions, or `type-` for types.
-*   in the meta.json file, the name of the output HTML file must be prefixed in the same way.
+-   the name of the HTML file you write must be prefixed with `function-` for functions, or `type-` for types.
+-   in the meta.json file, the name of the output HTML file must be prefixed in the same way.
 
 So the meta.json entry for a function would look like:
 
 ```
 "translateX": {
-    "baseTmpl": "tmpl/live-css-tmpl.html",
     "cssExampleSrc":
         "../../live-examples/css-examples/transforms/translate.css",
     "exampleCode":
-        "live-examples/css-examples/transforms/function-translateX.html",
+        "./live-examples/css-examples/transforms/function-translateX.html",
```

Comment:
It looks like the `cssExamplesSrc` line should also be updated, here and in the other spots in this file (see https://github.com/mdn/interactive-examples/pull/1174/files#diff-354f523859ee264d879542cab1eb4f71R4).